src/App.js

```diff
@@ -5,15 +5,19 @@ import './App.css'
 import Home from './scenes/Home/Home'
 import WeatherHourlyList from './scenes/WeatherHourlyList/WeatherHourlyList'
 import Login from './scenes/Login/Login'
+import { getToken } from './useToken'
```

Comment:
rename the file to `auth.js`. This is not a custom hook so should not have the prefix `use`

Reply:
![image](https://user-images.githubusercontent.com/62316127/125916868-fd3c3203-3fe4-4cdb-84fc-a4df57e7abe1.png)